src/shared-config/sharedRoutes.ts

```diff
@@ -1,10 +1,23 @@
-import { GenericRoute } from 'types/portal-config'
+import { GenericRoute, NestedRoute } from 'types/portal-config'
 
-// We handle the "normal" home route in App.tsx
-// This just gives us a fallback to use if we hit a failure case in the RouteResolver
-export const homeRoute: GenericRoute = {
-  to: '/Home',
-  isNested: false,
+const homeRoute: GenericRoute = {
+  path: '',
+  exact: true,
+  hideRouteFromNavbar: true,
+  synapseConfigArray: [
+    {
+      name: 'Header',
+      props: undefined,
+      isOutsideContainer: true,
+    },
+  ],
+}
+
```

Comment:
Deleted the home component, which just showed the Header component. Add it back as a (shared) route.